WeatherTodayTests/Sources/Specs/Supports/Extensions/Foundation/OptionalUnwrapSpec.swift

```diff
@@ -3,7 +3,7 @@ import Quick
 
 @testable import WeatherToday
 
-final class OptionalUnwrapSpec: QuickSpec {
+final class OptionalUnwrapSpec: QuickSpec{
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/ci-cd-sample-ios/blob/3df77319a1d3495943321a24797fc7d953905cbd/WeatherTodayTests/Sources/Specs/Supports/Extensions/Foundation/OptionalUnwrapSpec.swift#L6"></span>Opening braces should be preceded by a single space and on the same line as the declaration.<br />
<code>opening_brace</code> <code>OptionalUnwrapSpec.swift:6</code></td>
    </tr>
  </tbody>
</table>